stubs/python-dateutil/dateutil/tz/win.pyi

```diff
@@ -18,9 +18,9 @@ if sys.platform == "win32":
 
     class tzres:
         p_wchar: ClassVar[type[_Pointer[c_wchar]]]
-        def __init__(self, tzres_loc="tzres.dll"): ...
-        def load_name(self, offset): ...
-        def name_from_string(self, tzname_str: str): ...
+        def __init__(self, tzres_loc: str = "tzres.dll") -> None: ...
```

Comment:
It might be worth using a type that `ctypes.WinDLL()` accepts inside (`_NameTypes` is type alias from `ctypes`),
and attribute could be specified right away:
```suggestion
        tzres_loc: _NameTypes
        def __init__(self, tzres_loc: _NameTypes = "tzres.dll") -> None: ...
```